README.md

```diff
@@ -29,7 +29,7 @@ To create a plain blank project
 mvn archetype:generate^
  -DarchetypeGroupId=org.terasoluna.gfw.blank^
  -DarchetypeArtifactId=terasoluna-gfw-web-blank-archetype^
- -DarchetypeVersion=5.3.1.RELEASE
+ -DarchetypeVersion=5.4.0-SNAPSHOT
```

Comment:
@btkobayashirun 
SNAPSHOT? Really?

Reply:
I fixed it with daf26986fc6ee7cefa18a1ea34a20d9cb7b781d1